Sliver client crashes when fetching the task information of a previous process list task.

**Describe the bug**
The sliver client crashes when trying to fetch the task information of a previous process listing task. The sliver server will still be running but the client will crash with `panic: missing flag value: flag 'tree' not registered`.

**To Reproduce**
Steps to reproduce the behavior:
```
[*] Beacon 38b28cc6 SCARED_PHOTO - 192.168.122.241:49895 (DESKTOP-PC2FRCU) - windows/amd64 - Thu, 16 Feb 2023 13:33:15 EST

sliver > use 38b28cc6-2140-421a-ba37-2d19a883101d

[*] Active beacon SCARED_PHOTO (38b28cc6-2140-421a-ba37-2d19a883101d)

sliver (SCARED_PHOTO) > ps

[*] Tasked beacon SCARED_PHOTO (97e00d6c)

[+] SCARED_PHOTO completed task 97e00d6c

 Pid    Ppid   Owner                  Arch     Executable                    Session 
====== ====== ====================== ======== ============================= =========
 0      0                                      [System Process]                      
 4      0                                      System                                
 92     4                                      Registry                              
 352    4                                      smss.exe                              
 460    452                                    csrss.exe                             
 540    452                                    wininit.exe                           
 548    532                                    csrss.exe                             
 620    532                                    winlogon.exe                          
 676    540                                    services.exe                          
 696    540                                    lsass.exe                             
 784    676                                    svchost.exe                           
 796    620                                    fontdrvhost.exe                       
 804    540                                    fontdrvhost.exe               
... [snip] ...
 4324   3304   DESKTOP-PC2FRCU\User   x86_64   firefox.exe                           
 1724   784                                    WmiPrvSE.exe                          
 6136   4336   DESKTOP-PC2FRCU\User   x86_64   SCARED_PHOTO.exe                      
 1840   676                                    svchost.exe                           


⚠  Security Product(s): Windows Defender, Windows Smart Screen

sliver (SCARED_PHOTO) > tasks

 ID         State       Message Type   Created                         Sent                            Completed                     
========== =========== ============== =============================== =============================== ===============================
 97e00d6c   completed   Ps             Thu, 16 Feb 2023 13:33:24 EST   Thu, 16 Feb 2023 13:33:26 EST   Thu, 16 Feb 2023 13:33:26 EST 


sliver (SCARED_PHOTO) > tasks fetch 97e00d6c

+------------------------------------------------------+
| Beacon Task   | 97e00d6c-a7f2-4296-a4c3-a2a355c86d58 |
+---------------+--------------------------------------+
| State         | ✅ Completed                         |
| Description   | PsReq                                |
| Created       | Thu, 16 Feb 2023 13:33:24 EST        |
| Sent          | Thu, 16 Feb 2023 13:33:26 EST        |
| Completed     | Thu, 16 Feb 2023 13:33:26 EST        |
| Request Size  | 15 B                                 |
| Response Size | 11.7 KiB                             |
+------------------------------------------------------+

panic: missing flag value: flag 'tree' not registered

goroutine 1 [running]:
github.com/desertbit/grumble.FlagMap.Bool(0x5?, {0x17111f0, 0x4})
        github.com/desertbit/grumble@v1.1.1/flagmap.go:72 +0x105
github.com/bishopfox/sliver/client/command/processes.PrintPS({0xc00061d8d5, 0x7}, 0xc00030c730, 0x1, 0xc000744670, 0xc00030c500)
        github.com/bishopfox/sliver/client/command/processes/ps.go:141 +0x14d
github.com/bishopfox/sliver/client/command/tasks.renderTaskResponse(0xc0005a24d0, 0xc00030c500)
        github.com/bishopfox/sliver/client/command/tasks/fetch.go:571 +0xe9d
github.com/bishopfox/sliver/client/command/tasks.PrintTask(0xc0005a24d0, 0x1b9cf80?)
        github.com/bishopfox/sliver/client/command/tasks/fetch.go:146 +0x858
github.com/bishopfox/sliver/client/command/tasks.TasksFetchCmd(0xc0000d0720, 0xc00030c500)
        github.com/bishopfox/sliver/client/command/tasks/fetch.go:98 +0x3ef
github.com/bishopfox/sliver/client/command.BindCommands.func61(0xc00050c2a0?)
        github.com/bishopfox/sliver/client/command/commands.go:717 +0x45
github.com/desertbit/grumble.(*App).RunCommand(0xc0000fe620, {0xc000620980?, 0x14?, 0x1b85b90?})
        github.com/desertbit/grumble@v1.1.1/app.go:273 +0x19f
github.com/desertbit/grumble.(*App).runShell(0xc0000fe620)
        github.com/desertbit/grumble@v1.1.1/app.go:485 +0x4c6
github.com/desertbit/grumble.(*App).RunWithReadline(0xc0000fe620, 0xc00068d548)
        github.com/desertbit/grumble@v1.1.1/app.go:407 +0x570
github.com/desertbit/grumble.(*App).Run(0xc0000fe620)
        github.com/desertbit/grumble@v1.1.1/app.go:291 +0x125
github.com/bishopfox/sliver/client/console.Start({0x1bf6fa8?, 0xc0003dc270}, 0x1a042b0, 0x1a03c10, 0x0)
        github.com/bishopfox/sliver/client/console/console.go:145 +0x577
github.com/bishopfox/sliver/client/cli.StartClientConsole()
        github.com/bishopfox/sliver/client/cli/cli.go:100 +0x1a6
github.com/bishopfox/sliver/client/cli.glob..func1(0x25bd520?, {0x17109e0?, 0x0?, 0x0?})
        github.com/bishopfox/sliver/client/cli/cli.go:74 +0x73
github.com/spf13/cobra.(*Command).execute(0x25bd520, {0xc000118010, 0x0, 0x0})
        github.com/spf13/cobra@v1.6.1/command.go:920 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x25bd520)
        github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.6.1/command.go:968
github.com/bishopfox/sliver/client/cli.Execute()
        github.com/bishopfox/sliver/client/cli/cli.go:105 +0x25
main.main()
        github.com/bishopfox/sliver/client/main.go:43 +0x17
```

**Expected behavior**
The sliver client should return the task information along with the output of the `ps` command.

**Desktop (please complete the following information):**
 - Host OS: Kali Linux
 - Sliver Version:
 ```
 sliver > version

[*] Client v1.5.34 - d2a6fa8cd6cc029818dd8d9e4a039bdea8071ca2 - linux/amd64
    Compiled at 2023-02-07 11:30:45 -0500 EST
    Compiled with go version go1.20 linux/amd64


[*] Server v1.5.34 - d2a6fa8cd6cc029818dd8d9e4a039bdea8071ca2 - linux/amd64
    Compiled at 2023-02-07 11:30:44 -0500 EST
```
 
 **Additional Info**
 This is only for the beacon implant and the same bug affects both the Windows and Linux payloads.

--- a/client/command/tasks/fetch.go
+++ b/client/command/tasks/fetch.go
@@ -566,6 +566,7 @@ func renderTaskResponse(task *clientpb.BeaconTask, con *console.SliverConsoleCli
 				"overflow":      &grumble.FlagMapItem{Value: false},
 				"skip-pages":    &grumble.FlagMapItem{Value: 0},
 				"print-cmdline": &grumble.FlagMapItem{Value: true},
+				"tree":          &grumble.FlagMapItem{Value: false},
 			},
 		}
 		processes.PrintPS(beacon.OS, ps, true, ctx, con)

